test/extensions/filters/network/thrift_proxy/driver/generate_fixture.sh

```diff
@@ -70,7 +70,8 @@ if [[ "$OSTYPE" == "msys" ]]; then
         port=$(shuf -n 1 -i 49152-65535)
```

Comment:
This PR's title is a little bit confusing. Inside this while is the feature of retrying fetch port, and this pr just adds an output log. Maybe the title of pr is more appropriate to be called "enable outputting the retry fetch port log in windows integration test."